openslides_backend/action/actions/meeting/update.py

```diff
@@ -144,3 +153,79 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         if meeting_check:
             assert_belongs_to_meeting(self.datastore, meeting_check, instance["id"])
         return instance
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        # group A check
+        if any([field in instance for field in meeting_settings_keys]) and not has_perm(
+            self.datastore,
+            self.user_id,
+            Permissions.Meeting.CAN_MANAGE_SETTINGS,
+            instance["id"],
+        ):
+            raise MissingPermission(Permissions.Meeting.CAN_MANAGE_SETTINGS)
+
+        # group B check
+        if "present_user_ids" in instance and not has_perm(
+            self.datastore, self.user_id, Permissions.User.CAN_MANAGE, instance["id"]
+        ):
+            raise MissingPermission(Permissions.User.CAN_MANAGE)
+
+        # group C check
+        if (
+            "reference_projector_id" in instance
+            or any(
+                [field for field in instance if field.startswith("default_projector")]
```

Comment:
I don't like this; if we add a new field with the same prefix this can lead to unwanted results. Maybe solve it with a regex or validate again afterwards that only wanted fields were matched

Reply:
It is not needed, the field in the instance is always "default_projector_$_id". So a single check is enough.

Reply:
Ah, right.